README.md

```diff
@@ -2,61 +2,116 @@
 
```

Comment:
I am OK with these changes on the readme file but not confident enough about the changes suggested in the contributing guideline